src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/TaxonImage.orm.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <mapped-superclass name="Sylius\Component\Core\Model\TaxonImage" table="sylius_taxon_image">
+
+        <unique-constraints>
+            <unique-constraint columns="id,code" name="code_idx" />
```

Comment:
@michalmarcinkowski WDYT about this `unique-constraint` ?
I haven't seen any usage of this across sylius, is there any better way to enforce composite unique keys?
Because i don't think it's wise to make image_code unique across the whole table.